### PR TITLE
README updates etc.

### PR DESCRIPTION
**Pull request by blrhc**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -103,30 +119,30 @@ The list below shows the currently active roles available at our current [testne
...
 
-0.05
+0.10
 ```
 
-The number - 0.05 cents per block - seems a bit underwhelming, but validation requires little effort for the user after setup, and blocks come in every 6 seconds. With armv7 binaries or low-end VPS nodes, it should be cheap to run!
+The number - 0.10 cents per block - seems a bit underwhelming, but validation requires little effort for the user after setup, and blocks come in every 6 seconds. With armv7 binaries or low-end VPS nodes, it should be cheap to run!
 
 ### Payouts
 `Validators` must include their [XMR address](#how-it-works) in the `memo` of their `controller` key.
```
Comment (bwhm) [inline README.md]:
```suggestion
`Validators` must include their [XMR address](#how-it-works) in the `memo` of their `stash` key.
```

File: roles/content-creators/README.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+First, visit the `Media` tab in the sidebar. You can then click on `New channel` in the top navigation bar to make your
+first channel. Complete the form with the details for your channel.
+
+Your channel will be `Public` by default. To upload content simply navigate to your channel and click the `Upload Video` button. From there you can upload video files before filling in their metadata and submitting them into the content directory.
+
+## Content Restrictions
+It is very important that you do not upload illegal or copyrighted content on our testnets. This will result in a disqualification from payouts. It will also result in the takedown of content, potentially slashing of funds, and the deletion of your channel. Multiple spam uploads which represent a burden to moderate for the `Content Curators` may also be penalized and result in deductions on payouts due for qualifying content uploads on your content creator profile.
+
+We will provide `Content Creators` with a list of sources for safe and non-copyrighted material which can be uploaded as part of this testnet and will qualify for the payments highlighted [here](https://github.com/blrhc/helpdesk/tree/rome#incentives-3).
```
Comment (bwhm) [inline roles/content-creators/README.md]: Could you add a link to either the existing, or a new issue, containing safe material?

If not, this should go away!

File: roles/content-creators/README.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+requires no extra software. Everything can be done in the browser, by going [here](http://testnet.joystream.org).
+
+## Get Started
+If you want to upload content as a `Content Creator`, you need to become a `Member`. Instructions for this 
+can be found [here](https://github.com/JoyStream/helpdesk/#get-started).
+
+## Uploading Content
+Once you are a member, it is extremely simple to get started uploading content to the platform.
+
+First, visit the `Media` tab in the sidebar. You can then click on `New channel` in the top navigation bar to make your
```
Comment (bwhm) [inline roles/content-creators/README.md:35]: Add that we are currently experiencing some issues with content uploading in chromium based browsers!

File: README.md [hunk 4]
```diff
@@ -48,10 +56,18 @@
...
+  - [Content Curators](#content-curators)
+    - [Description](#description-4)
+    - [Incentives](#incentives-4)
+    - [Payouts](#payouts-4)
+  - [Builders and Bug Reporters](#builders-and-bug-reporters)
+    - [Description](#description-5)
+    - [Incentives](#incentives-5)
+    - [Payouts](#payouts-5)
 - [Other Features and Future Roles](#other-features-and-future-roles)
   - [Upload Content](#upload-content)
```
Comment (bwhm) [inline README.md]: Remove!

**Review by bwhm — CHANGES_REQUESTED** [review]
Nice! Just some minor tweaks here...

**Comment by blrhc** [comment]
@bwhm Ok, made fixes requested :)

**Pull request merged**